### DISCUSSION TITLE
[Projects] Automatically create the next iteration

### DISCUSSION THREAD

**Discussion by archon810** [community/community#27908] [Projects and Issues]
When I first created the Iterations field in my Github project, 3 iterations were auto created, and I thought once one iteration closes, it'll top off with a new one automatically.

This doesn't happen, but it would be great if there was a setting for it. "Auto create next iteration" on/off.

**Comment by jfullstackdev** [comment]
is it a coding problem you are referring to ? you can post the code here so we can take a look

**Reply by archon810** [reply]
I'm referring to the Iterations capability of the new Github Projects.

**Comment by ralf-vogel** [comment]
It would be great to have some kind of auto-generation for the upcoming iterations. If the iterations are typically two weeks long, they should just be generated automatically. Ideally with an easy option to shift them - like a board where you can just insert a break or adjust the length of an iteration and the following iterations are automatically adjusted.

**Comment by skeet70** [comment]
I just assumed this worked, that iterations with the interval you configured were automatically created. And since theres a `@next` filter for iterations, I assumed it would always be creating a next automatically. If it doesn't that should _really_ be a feature.
